pkg/kubectl/autoscale.go

```diff
@@ -38,6 +39,7 @@ func (HorizontalPodAutoscalerV1Beta1) ParamNames() []GeneratorParam {
 		{"scaleRef-kind", false},
 		{"scaleRef-name", false},
 		{"scaleRef-apiVersion", false},
+		{"scaleRef-apiGroup", false},
```

Comment:
and I would actually call it `apiVersion`, parallel to the wire format for the request, and the treatment of apiVersion in all typemeta objects. There is a parse function that will take a string and return an unversioned.GroupVersion or an err